www/main/navigator/misc/components/MiscHeader.jsx

```diff
@@ -35,6 +36,16 @@ export const MiscHeader = () => {
           </span>
         </i>
       </a>
+      <a
+        className={`misc-button ${isFav ? 'misc-active' : ''}`}
+        onClick={event => SetOpenTab(event)}
+      >
+        <i className="fa fa-heart">
+          <span className="Icon-label" key="Favorite">
+            Favorite
```

Comment:
this should come from i18n veerji

Reply:
It's just one word labels, are we sure to use i18n here? None of the option in misc header is coming from i18n. So I continued this new tab is same way.

Reply:
Just saw we are using one word strings in i18n at other places too, updating this too